src/modlunky2/mem/state.py

```diff
@@ -203,6 +203,14 @@ def hud_flags(self):
         offset = self._offset + 0xA10
         return HudFlags(self._proc.read_u32(offset))
 
+    @property
+    def money_shop_total(self):
```

Comment:
do you know if this state counts when you lose money from monkies/leprechauns?

Reply:
I hadn't investigated before, but... yes, leprechauns send you an invoice for the money they stole. Updated docstring